internal/distro/rhel85/pipelines.go

```diff
@@ -952,6 +957,17 @@ func qemuPipeline(inputPipelineName, inputFilename, outputFilename, format, qcow
 	return p
 }
 
+func bootloaderConfigStage(t *imageType, partitionTable disk.PartitionTable, kernel *blueprint.KernelCustomization, kernelVer string) *osbuild.Stage {
+	if t.arch.name == s390xArchName {
+		return osbuild.NewZiplStage(new(osbuild.ZiplStageOptions))
```

Comment:
I would prefer `osbuild.NewZiplStageOptions()` but not a big deal. :)

Reply:
Agreed.  I'm working on adding a bunch of functions like that for all stages.